src/server/game/Movement/MovementGenerators/PointMovementGenerator.cpp

```diff
@@ -142,8 +142,6 @@ void EffectMovementGenerator::Finalize(Unit* unit)
     {
         if (Unit* victim = unit->GetVictim())
             unit->GetMotionMaster()->MoveChase(victim);
-        else
-            unit->GetMotionMaster()->Initialize();
```

Comment:
any idea about what could this break ? also why do you need to remove this to fix the smart script ?